### PR TITLE
s3: add petabox.io to s3 providers list

### PR DESCRIPTION
**Pull request by cadet354**
#### What is the purpose of this change?
Add [Petabox.io](https://petabox.io) to S3 providers. I've ran rclone's integration tests successfully.

#### Was the change discussed in an issue or in the forum before?
No.

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [X] I have added tests for all changes in this PR if appropriate.
- [X] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [X] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: backend/s3/s3.go [hunk 1]
```diff
@@ -2717,6 +2764,7 @@ func s3Connection(ctx context.Context, opt *Options, client *http.Client) (*s3.S
 		awsConfig.WithEndpointResolver(r)
 	}
 
+	awsConfig.WithLogLevel(aws.LogDebug)
```
Comment (ncw) [inline backend/s3/s3.go:2767]: I think this isn't supposed to be in this commit.

Maybe we should have an `--s3-debug` flag which turns it on though? Something for a followup PR?

**Review by ncw — COMMENTED** [review]
This looks nice!

I think there is a stray debug in the code which I've pointed out.

This is missing the docs though - check out 0a8b1fe5dee556c375af82d1eff264e4e3b25da5 for an idea of the docs files which need patching.

Thank you

File: backend/s3/s3.go (hunk 1 above)
Comment (cadet354) [reply backend/s3/s3.go]: You'r right, i removed that line.

**Comment by ncw** [comment]
If you look in the commit I linked above you'll see the docs you need to edit. All the other docs are built and you don't need to do this for the PR - it gets done at the release time. I suggest you don't bother adding the autogenerated docs in `s3.md` these get added at release time too and I don't really need them as part of the PR.

If you want to check to see what the docs look like rendered, then use `make serve` (make sure you have `hugo` installed) and look at the web pages it serves.

```
$ git show --stat  0a8b1fe 
commit 0a8b1fe5dee556c375af82d1eff264e4e3b25da5
Date:   Tue Dec 6 15:55:23 2022 +0330

    s3: add Liara LOS to provider list

 README.md              |   1 +
 backend/s3/s3.go       |  37 +++++++++++++++++++++++++++++++++----
 docs/content/_index.md |   1 +
 docs/content/s3.md     | 118 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++--------
 4 files changed, 145 insertions(+), 12 deletions(-)
```

**Comment by cadet354** [comment]
@ncw Hi! I updated _index.md and README.md, but not s3.md as you suggested. When i ran `make serve` i see Petabox on the main page:
image
The `Home` button works well, but the `Config` leads to `/s3` page, and Petabox does not appear in the list of supported s3 providers on that page.
image
I also tried to launch the website after `make doc`, but i get the same results. You sure i don't need to modify the `s3.md`?

Thanks!

**Comment by cadet354** [comment]
@ncw Hi! I edited s3.md, now everything works fine! Please take a look

**Review by ncw — APPROVED** [review]
That looks great now - thank you :-)

I'll merge now.

**Comment by cadet354** [comment]
Thanks!

**Pull request merged**